less/mixins.less

```diff
@@ -1 +1,16 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.flexbox(){
+    display: flex;
+    flex-direction: center;
```

Comment:
`flex-direction: center;` isn't a valid value for flex-direction (valid values are row, column, row-reverse, column-reverse)

I'd also turn this into a prarametric mixin, like this:
```
.flexbox(@justify:center, @align:center){
    display: flex;
    // flex-direction: center;
    justify-content: @justify;
    align-items: @align;
}
```
If you call `.flexbox()` just like normal, then the centered styles you have will automatically be added. Check out my comment in `navigation.less` to see how I use it '' '